compiler/passes/codegen.cpp

```diff
@@ -51,6 +51,24 @@ int      gMaxVMT    = -1;
 int      gStmtCount =  0;
 
 
+// ensure these two produce consistent output
+std::string zlineToString(BaseAST* ast) {
+  return "/* ZLINE: " + numToString(ast->linenum())
+         + " " + ast->fname() + " */\n";
+}
+void zlineToFileIfNeeded(BaseAST* ast, FILE* outfile) {
+  if (printCppLineno)
+    fprintf(outfile, "/* ZLINE: %d %s */\n", ast->linenum(), ast->fname());
```

Comment:
This looks like it could use zlineToString to make a string to put into outfile.